### DISCUSSION TITLE
How to handle Missing data

### DISCUSSION THREAD

**Discussion by sege01** [zalandoresearch/pytorch-ts#68] [Q&A]
Hello,

Thank you for your awesome work. 
I was trying to test TransformerTempFlowEstimator on my data. I needed some help in understanding how one can handle missing data in the case of demand forecasting. For example, there could be some products for which data is not available like snow boots are not available during summer. So, there is no data for snow boots for summer. How this kind of data with a gap can be handled?

Your suggestions are highly appreciable.

Thank you.

**Comment by sege01** [comment]
Resolved

**Reply by LemonCANDY42** [reply]
Can you tell me how you solved it?

**Reply by LemonCANDY42** [reply]
> Can you tell me how you solved it?

like used zero?
